package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "js-cookie",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
There is no need to increase the version in this commit, this is done when a new version is released.